android/build.gradle

```diff
@@ -132,7 +132,8 @@ dependencies {
   api 'com.facebook.react:react-native:+'
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.3.1"
-  implementation 'com.stripe:stripe-android:19.3.+'
+  implementation 'com.stripe:stripe-android:20.0.+'
+  implementation 'com.stripe:connections:20.0.+'
```

Comment:
Undecided on if this is included by default or we require users to add it

Reply:
yep- my question here is if the added size of the connections library is large enough to make users add this to their build.gradle manually. Even small additional steps like that can really cause issues for RN users. And then they're in charge of keeping that version up-to-date with the version of `com.stripe:stripe-android` we use is stripe-react-native internally. So I lean towards including it in our own library so users don't have to think about it.